compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/model/BIRAbstractInstruction.java

```diff
@@ -41,4 +41,6 @@ public InstructionKind getKind() {
     }
 
     public abstract BIROperand[] getRhsOperands();
+
+    public abstract void setRhsOperands(BIROperand[] operands);
```

Comment:
Is there any reason for adding this as an abstract method? Since most of the places we are doing nothing with this, can we override in the necessary places only?

Reply:
To be consistent with `getRhsOperands` we can keep it as an abstract method.